mlrun/model.py

```diff
@@ -25,6 +25,16 @@
 from .utils import dict_to_yaml, get_in, dict_to_json, get_artifact_target
 
 
+class ResourceSchema:
+    FeatureSet = "fset"
+    FeatureVector = "fvec"
+    Artifact = "store"
+
+    @classmethod
+    def is_resource(cls, kind):
+        return kind in [cls.Artifact, cls.FeatureSet, cls.FeatureVector]
```

Comment:
What is this ?

Reply:
its for the next PR, define the special prefixes (url schema) for feature store and artifact store data urls